docs/configuration.rst

```diff
@@ -472,3 +477,68 @@ Optional Settings
 All other settings for Connect like security, monitoring interceptors, producer and consumer overrides can be passed to the Docker images as environment variables. The names of these environment variables are derived by replacing ``.`` with ``_``, converting the resulting string to uppercase and prefixing it with ``CONNECT_``. For example, if you need to set ``ssl.key.password``, the environment variable name would be ``CONNECT_SSL_KEY_PASSWORD``.
 
 The image will then convert these environment variables to corresponding Connect config variables.
+
+-----------
+KSQL Server
+-----------
+
+The KSQL Server image uses variables prefixed with ``KSQL_`` with an underscore (``_``) separating each word instead of periods. As an example, the following command runs the KSQL Server passing in the details of the underlying Kafka cluster to use, the port to listen on, the location where persistent data should be store, and service id of the KSQL server pool, and so on.
+
+.. sourcecode:: bash
+
+  docker run -d \
+    -p 127.0.0.1:8088:8088
+    -e KSQL_BOOTSTRAP_SERVERS=localhost:9092 \
+    -e KSQL_LISTENERS=http://0.0.0.0:8088 \
+    -e KSQL_KSQL_SERVICE_ID=ksql_test_service_id_ \
+    -e KSQL_KSQL_STREAMS_REPLICATION_FACTOR=3 \
+    -e KSQL_KSQL_SINK_REPLICAS=3 \
+    -e KSQL_KSQL_STREAMS_STATE_DIR=/mnt/data/ksql-server-data \
```

Comment:
We should clarify the paths for Docker beginners, this is so easy to get wrong (anecdotal evidence: while typing the suggestion below I first mixed up the paths, doh, even though I was focused on 'the path problem').

I'd suggest to change
```
-e KSQL_KSQL_STREAMS_STATE_DIR=/mnt/data/ksql-server-data \
-v /mnt/ksql-server/data:/mnt/data/ksql-server-data/ \
```

to

```
-e KSQL_KSQL_STREAMS_STATE_DIR=/path/in/container/ksql-server-data \
-v /path/on/docker/host/ksql-server-data:/path/in/container/ksql-server-data \
```